cmd/buildx/main.go

```diff
@@ -40,6 +40,9 @@ func init() {
 }
 
 func runStandalone(cmd *command.DockerCli) error {
+	stopProfiles := setupDebugProfiles(context.TODO())
```

Comment:
Why not call this from `main()` directly to avoid duplicate calls.

Reply:
This can also be a single defer line

Reply:
This is just a style thing. I find this easier to read.